app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,38 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.domain.Identifier;
+import com.codesoom.assignment.errors.LoginDataNotMatchedException;
+import com.codesoom.assignment.utils.JwtUtil;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+/**
+ * 인증 token 정보를 관리한다.
+ */
+@Service
+@RequiredArgsConstructor
+public class AuthenticationService {
+    private final UserRepository userRepository;
+    private final JwtUtil jwtUtil;
+
+    /**
+     * 로그인 정보를 조회해서 토큰을 발급한다.
+     *
+     * @param identifier 이메일과 비밀번호가 포함된 로그인 정보
+     * @return 토큰
+     * @throws LoginDataNotMatchedException 아이디 혹은 비밀번호가 유효하지 않다.
+     */
```

Comment:
이렇게 하면 메소드 내용을 javadoc으로부터 적절히 숨길 수 있겠죠.
```suggestion
    /**
     * 사용자를 식별하여 인증하고, 인증 토큰을 발급한다.
     *
     * @param identifier 사용자 식별 정보
     * @return 인증 토큰
     * @throws LoginDataNotMatchedException 인증에 실패한 경우
     */
```

Reply:
단순히 제안해주신 내용을 작성할게 아니라 어떤식으로 수정을 했는지 고민을 하니 메소드의 내부 로직이 어떤지 어떤 파라미터를 받는지 구구절절히 설명하기보다는 목적에 충실해야 한다는 점이 핵심인 것 같습니다. 

식별자에 아이디와 비밀번호가 아닌 다른 정보를 식별자로 사용할 수 있고, 아이디 혹은 비밀번호가 틀린것이 아닌 다른 경우로 인증에 실패한 경우에 예외가 발생할수도 있으니 좀 더 포괄적으로 주석을 작성할 수 있도록 노력해야겠습니다.

Reply:
깨달음을 얻으셨군요. 기쁩니다. 👍 👏